### PR TITLE
Yifei/hack mshReader to save msh with tag

### PR DESCRIPTION
**Pull request by ahighmoon**

### CODE REVIEW

File: src/wmtk/io/MeshReader.cpp [hunk 1]
```diff
@@ -1,9 +1,8 @@
 #include "MeshReader.hpp"
 
-#include <wmtk/Mesh.hpp>
+// #include <wmtk/Mesh.hpp>
```
Comment (daniel-zint) [inline src/wmtk/io/MeshReader.cpp:3]: remove commented out code

File: src/wmtk/io/MshReader.cpp [hunk 2]
```diff
@@ -21,6 +20,22 @@ std::shared_ptr<Mesh> MshReader::read(const std::filesystem::path& filename, boo
...
     if (get_num_tets() > 0) {
+        // std::cout << "Tetmesh!" << std::endl;
+        // std::cout << get_num_tet_vertices() << std::endl;
+        // std::cout << get_num_tets() << std::endl;
+        // std::cout << "element_data_size = " << m_spec.element_data.size() << std::endl;
+        // std::cout << "node_data_size = " << m_spec.node_data.size() << std::endl;
+        for (const auto& data : m_spec.element_data) {
+            const auto& int_tags = data.header.int_tags;
+            // std::cout << "element_data_int_tags = " << int_tags.size() << std::endl;
+            std::cout << data.header.string_tags.front() << std::endl;
```
Comment (daniel-zint) [inline src/wmtk/io/MshReader.cpp:31]: please remove all `std::cout` statements

File: src/wmtk/io/MshReader.cpp [hunk 3]
```diff
@@ -64,6 +79,8 @@ std::shared_ptr<Mesh> MshReader::read(const std::filesystem::path& filename, boo
 
         auto tmp = std::make_shared<TetMesh>();
         tmp->initialize(S);
+        // auto attr_acc =
+        extract_element_attribute(tmp, "in/out", 3);
```
Comment (daniel-zint) [inline src/wmtk/io/MshReader.cpp:83]: This must not be hard coded.

File: src/wmtk/io/MshReader.hpp [hunk 4]
```diff
@@ -12,7 +15,7 @@ class MshReader
 public:
     std::shared_ptr<Mesh> read(const std::filesystem::path& filename, const bool ignore_z = false);
 
-private:
+    // private:
```
Comment (daniel-zint) [inline src/wmtk/io/MshReader.hpp:18]: pretty sure that should be private

**Review by daniel-zint — COMMENTED** [review]
some minor things

**Review by mtao — APPROVED** [review]
lets start cleaning up all the comments in the code

**Pull request merged**